src/frontend/komponenter/Fagsak/Brevmottaker/LeggTilEndreBrevmottakerModal.tsx

```diff
@@ -153,10 +229,10 @@ export const LeggTilEndreBrevmottakerModal: React.FC<Props> = ({ brevmottakerId
                             loading={skjema.submitRessurs.status === RessursStatus.HENTER}
                             disabled={skjema.submitRessurs.status === RessursStatus.HENTER}
                             onClick={() =>
-                                lagreBrevmottakerOgOppdaterState(brevmottakerId, lukkModal)
+                                lagreBrevmottakerOgOppdaterState(brevmottakerTilEndring, lukkModal)
                             }
                         >
-                            Legg til mottaker
+                            {brevmottakerTilEndring ? 'Lagre endringer' : 'Legg til'}
```

Comment:
Hvis man åpner modalen for å endre en brevmottaker og skjemaet allerede har gyldige verdier kommer fortsatt `valideringErOk()` til å være falsy og dermed vil knappen være secondary. Valideringen kjøres ved endring av feltene, men du kan også trigge den med `validerAlleSynligeFelter()` hvis du vet at skjemaet allerede er fylt med gyldige verdier fra backend, så blir knappen primary :smiling:

Reply:
La merke til det da vi jobbet med refusjon EØS https://github.com/navikt/familie-ba-sak-frontend/pull/2641/commits/06374bd56cee13acf5e8296db8db6f58f6714ddf

Reply:
Det kan med fordel legges inn, ja 👍